pkgs/development/libraries/openldap/default.nix

```
@@ -54,19 +56,20 @@ stdenv.mkDerivation rec {
     "moduledir=$(out)/lib/modules"
   ];
 
-  # 1. Fixup broken libtool
-  # 2. Libraries left in the build location confuse `patchelf --shrink-rpath`
+  # 1. Libraries left in the build location confuse `patchelf --shrink-rpath`
   #    Delete these to let patchelf discover the right path instead.
   #    FIXME: that one can be removed when https://github.com/NixOS/patchelf/pull/98
   #    is in Nixpkgs patchelf.
+  # 2. Fixup broken libtool for openssl and cyrus_sasl respectly if they are not
+  #    overridden to be null.
```

```suggestion
  # 2. If build with withCyrusSasl fixup broken libtool
```

Or something similar but we no longer fixup openssl so the comment is wrong right now.

---

Fixed the comment and re added the openssl fix which I removed erroneously yesterday.